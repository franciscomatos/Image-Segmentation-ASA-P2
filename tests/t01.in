5 5

8 7 9 9 7
6 2 2 8 7
9 1 2 1 8
2 1 3 1 7
1 3 2 1 9

2 1 2 3 2
1 9 9 1 3
1 7 7 9 3
8 7 9 7 2
7 9 8 9 1

8 9 7 6
1 9 0 8
1 8 9 2
8 7 9 1
9 8 7 2

8 2 1 7 9
7 8 7 1 8
2 8 7 7 8
9 7 8 9 8
