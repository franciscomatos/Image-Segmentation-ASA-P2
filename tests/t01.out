57

C C C C C 
C P P C C 
C P P P C 
P P P P C 
P P P P C 
